spec/controllers/registrations_controller_api_spec.rb

```diff
@@ -81,13 +81,13 @@ def register_user_with_params( params = {} )
   end
 
   it "should assign a user to a site" do
-    @site = Site.make!( url: "test.host" ) # hoping the test host is the same across platforms...
+    @site = Site.make!( url: "http://test.host" ) # hoping the test host is the same across platforms...
```

Comment:
Assuming these should be URLs and not just the domains. Caused an issue `both URI are relative` which assuming all records in `sites` have proper URLs, shouldn't be occurring.

Reply:
Yeah, these should be URLs. Not sure why that was working.